Level 1

pacsal opened the pull request:
This solution uses simple HTML and Javascript on level 1 & 2, this is completed level 2 solution. It can be written in any code editor and run on any browser.
No need of a local server of any plugging.

Link to source codes: https://github.com/pacsal/TheShopkeeperChangeChallenge

The following are the screenshots of the source codes and its UI output. Enjoy!!
=> HTML part
![image](https://user-images.githubusercontent.com/12777378/69619040-fbb8b800-104b-11ea-895f-9bbd588fb4f3.png)

=>JS part
![image](https://user-images.githubusercontent.com/12777378/69619175-39b5dc00-104c-11ea-9f26-7d4c452ab271.png)

=>UI part
![image](https://user-images.githubusercontent.com/12777378/69619397-8dc0c080-104c-11ea-9165-92f6cab2548d.png)

pacsal committed:
Level 1

Use simple html and javascript on level 1

Davisy commented:
# Davis source code LEVEL 1 submission

`def shoopKeeperApp():
    shoppInput = int(input("The amount due as change:",))
    
    #calculate number of 1000 notes
    if shoppInput >= 1000:
        opp_1000 = shoppInput/1000
        remain_1000 = opp_1000 % 1000 
        
        #calculate number of 1000 notes
        if (opp_1000 >= 1):
            n_1000 = int(opp_1000)
        else:
            n_1000 = None 
     
    #calcualte number of 500 notes
    elif (shoppInput >=500):
        opp_500 = shoppInput / 500 
        
        #calculate number of 500 notes
        if (opp_500 >= 1):
            n_500 = int(opp_500)
            remain_500 = opp_500 % 500
        else:
            n_500 = None 
            
    elif (remain_1000 != 0 and remain_1000 >= 500):
        opp_500 = shoppInput / 500 
        
        #calculate number of 500 notes
        if (opp_500 >= 1):
            n_500 = int(opp_500)
            remain_500 = opp_500 % 500    
                        
        else:
            n_500 = None 
                        
    #calcualte number of 100 notes
    elif (shoppInput >= 100):
        opp_100 = shoppInput / 100 
        
        #calculate number of 100 notes
        if (opp_100 >= 1):
            n_100 = int(opp_100)
            remain_100 = opp_100 % 100
        else:
            n_100 = None 
            
    elif (remain_500 != 0 and remain_500 >= 100):
        opp_100 = shoppInput / 100 
        
        #calculate number of 100 notes
        if (opp_100 >= 1):
            n_100 = int(opp_100)
            remain_100 = opp_100 % 100    
                        
        else:
            n_100 = None 
            
        #calcualte number of 50 notes
    elif (shoppInput >= 50):
        opp_50 = shoppInput / 50 
        
        #calculate number of 50 notes
        if (opp_50 >= 1):
            n_50 = int(opp_50)
            remain_50 = opp_50 % 50
        else:
            n_50 = None 
            
    elif (remain_100 != 0 and remain_100 >= 50):
        opp_50 = shoppInput / 50 
        
        #calculate number of 100 notes
        if (opp_50 >= 1):
            n_50 = int(opp_50)
            remain_50 = opp_50 % 50    
                        
        else:
            n_50 = None 
            
            
    #calcualte number of 20 coins
    elif (shoppInput >= 20):
        opp_20 = shoppInput / 20 
        
        #calculate number of 20 notes
        if (opp_20 >= 1):
            c_20 = int(opp_20)
            remain_20 = opp_20 % 20
        else:
            c_20 = None 
            
    elif (remain_50 != 0 and remain_50 >= 20):
        opp_20 = shoppInput / 20 
        
        #calculate number of 100 notes
        if (opp_20 >= 1):
            c_20 = int(opp_20)
            remain_20 = opp_20 % 20    
                        
        else:
            c_20 = None
            
            
            
    #calcualte number of 10 coins
    elif (shoppInput >= 10):
        opp_10 = shoppInput / 10 
        
        #calculate number of 10 notes
        if (opp_10 >= 1):
            c_10 = int(opp_10)
            remain_10 = opp_10 % 10
        else:
            c_10 = None 
            
    elif (remain_20 != 0 and remain_20 >= 10):
        opp_10 = shoppInput / 10 
        
        #calculate number of 10 
        if (opp_10 >= 1):
            c_10 = int(opp_10)
            remain_10 = opp_10 % 10    
                        
        else:
            c_10 = None

            
    #calcualte number of 5 coins
    elif (shoppInput >= 5):
        opp_5 = shoppInput / 5 
        
        #calculate number of 10 notes
        if (opp_5 >= 1):
            c_5 = int(opp_5)
            remain_5 = opp_5 % 5
        else:
            c_5 = None 
            
    elif (remain_10 != 0 and remain_10 >= 5):
        opp_5 = shoppInput / 5 
        
        #calculate number of 5 
        if (opp_5 >= 1):
            c_5 = int(opp_5)
            remain_5 = opp_5 % 5    
                        
        else:
            c_5 = None

    #calcualte number of 1 coins
    elif ( shoppInput >= 0 and shoppInput < 5 ):
        c_1 = shoppInput 
            
    elif (remain != 0 and remain_5 >= 1 and  remain_5 < 5):
        c_1 = remain_5   
                        
    else:
        c_1 = None
        
        
    print("n_1000: {}".format(n_1000))
    print("n_500: {}".format(n_500))
    print("n_100: {}".format(n_100))
    print("n_50: {}".format(n_50))
    print("c_20: {}".format(c_20))
    print("c_10: {}".format(c_10))
    print("c_5: {}".format(c_5))
    print("c_1: {}".format(c_1))
        
        
    
    
    
shoopKeeperApp() `

pacsal committed:
Level 2

ShopKeeper/customer can shop the items and get a bill at the end.

pacsal committed:
Level 2

Customer/ShopKeeper can select items shoped, and amount billed will be filled automaticall.

ShopeKeeper will put amount given by customer and press calculate.

Shopkeeper will get exactly change to be issued and how many coins or note should be issued.

//this is done by javascript and html5
